bolt-contracts/script/holesky/operators/RegisterEigenLayerOperator.s.sol

```diff
@@ -19,9 +20,27 @@ contract RegisterEigenLayerOperator is Script {
         uint256 expiry;
     }
 
-    function run() public {
+    function S01_depositIntoStrategy() public {
         uint256 operatorSk = vm.envUint("OPERATOR_SK");
 
+        IStrategyManager strategyManager = _readStrategyManager();
+
+        string memory json = vm.readFile("config/holesky/operators/eigenlayer/depositIntoStrategy.json");
+
+        IStrategy strategy = IStrategy(vm.parseJsonAddress(json, ".strategy"));
+        IERC20 token = IERC20(vm.parseJsonAddress(json, ".token"));
+        uint256 amount = vm.parseJsonUint(json, ".amount");
+
+        vm.startBroadcast(operatorSk);
+        // Allowance must be set before depositing
+        token.approve(address(strategyManager), amount);
+        strategyManager.depositIntoStrategy(strategy, token, amount);
+        console.log("Successfully run StrategyManager.depositIntoStrategy");
+        vm.stopBroadcast();
+    }
+
+    function S02_registerIntoBoltAVS() public {
```

Comment:
Can we remove the `S0_1 prefixes on the functions, we don't have it anywhere else. Unless you have a good reason

Reply:
The reasons are the following:
- having a single file for each script would resulting in copying over the same utilities functions over and over again, so I put everything into one file, `RegisterEigenLayerOperator.s.sol`
- given this scripts, it makes to ask whether there is an order with which you should run them. This allows you to self-document the script file and I took inspiration from it from Forge's scripts best practices https://book.getfoundry.sh/tutorials/best-practices#scripts